dipy/segment/bundles.py

```diff
@@ -67,9 +73,106 @@ def bundle_adjacency(dtracks0, dtracks1, threshold):
     return res
 
 
+def ba_analysis(recognized_bundle, expert_bundle, threshold=2.):
+    """ Calculates bundle adjacency score between two given bundles
+
+    Parameters
+    ----------
+    recognized_bundle : Streamlines
+    expert_bundle : Streamlines
+    threshold: float (default 2)
+
+    References
+    ----------
+    .. [Garyfallidis12] Garyfallidis E. et al., QuickBundles a method for
+                        tractography simplification, Frontiers in Neuroscience,
+                        vol 6, no 175, 2012.
+    """
+
+    recognized_bundle = set_number_of_points(recognized_bundle, 20)
```

Comment:
Maybe make the resampling resolution (here hard-coded as 20) a key-word argument defaulting to 20 instead?